.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[alias]
+# equivalent to cargo test with no-color in this project
+r = "run -- --features no-color"
+t = "test --features no-color"
```

Comment:
I would prefer:

```toml
run-without-color = "run -- --features no-color"
test-without-color = "test --features no-color"
```

or 

```toml
run-colorless = "run -- --features no-color"
test-colorless = "test --features no-color"
```

Here is OK, but not where you use it. If you read `cargo r`, you have to go and read the alias (if you know that it can be a custom cargo command).

Reply:
> Do you think we should do the same (with cargo)?

I didn't think of that. Thx for pointing it out. I know the `--color` arguement in cargo, and I think we should adhere that.
I'll put forward a new PR for it.


> I would prefer:
> 
> ...
> 
> Here is OK, but not where you use it. If you read `cargo r`, you have to go and read the alias (if you know that it can be a custom cargo command).

To be clear, `cargo r` and `cargo t` , the **builtin cargo command aliases**, are meant for our project, not for the ultimate users. A user is only aware of `cargo pretty-test`.
So I change them here just to run tests conveniently in dev because
* `cargo r` in the project means running `./target/debug/cargo-pretty-test` by default cargo, and since `cargo-pretty-test` runs `cargo test`
    * `cargo r` also means `cargo test` conceptually (aka running tests via cargo)
* we want testing to be no-color **always**, the easiest way is passing `--feature no-color` whenever running tests
* so making (and overriding) aliases here is for easy use in the project during dev